tasks1.js

```diff
@@ -0,0 +1,118 @@
+// 1
+function convertMinutesToHours(minutes) {
+  return (Math.floor(minutes / 60)) + ':' + (minutes % 60);
+}
+console.log(convertMinutesToHours(75));
+
+
+// 2
+function isTextUppercase(text) {
+  return text.toUpperCase() === text;
+}
+console.log(isTextUppercase('START'));
+
+
+// 3
+function isDivisible(number1, number2, number3) {
```

Comment:
Those argument names are a bit too generic and don't describe what those numbers mean.

The first number is the number we divide.
The second number is the first number we divide by.
The third number is the second number we divide by.